web/styles/components.css

```diff
@@ -96,7 +96,13 @@ a.no-underline:hover {
 i.zulip-icon.zulip-icon-bot {
     color: var(--color-icon-bot);
     vertical-align: top;
-    padding: 0 2px;
+    padding: 0 0 0 5px;
```

Comment:
This likely affects every instance of `.zulip-icon-bot` in the interface; have you grepped to make sure that it's not breaking other presentations, e.g., in message headers?